### PR TITLE
feat: add mitosis testnet

### PR DESCRIPTION
**Pull request by JunayedMh**
### Description
add mitosis testnet
docs : https://docs.mitosis.org/docs/learn/game-of-mito/setup
minified by: https://jakearchibald.github.io/svgomg/

### Testing
yes

### CODE REVIEW

File: chains/mitosistestnet/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://testnet.mitosiscan.xyz/api
+    family: routescan
```
Comment (JaunePomme) [inline chains/mitosistestnet/metadata.yaml:4]: Not sure about that family name. I think it would be more blockscout or something else

**Comment by JaunePomme** [comment]
Overall looks good to me, besides the family name, and dunno if you minimified the svg (https://jakearchibald.github.io/svgomg/).

File: chains/mitosistestnet/metadata.yaml (hunk 1 above)
Comment (JunayedMh) [reply chains/mitosistestnet/metadata.yaml:4]: Yeah it is routerscan
Chk on the explorer

Comment (k4ctuss) [reply chains/mitosistestnet/metadata.yaml:4]: you should make family name to "other" cause yours isn't propose by the CLI

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**